charts/trusted-attestation-controller/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+name: trusted-attestation-controller
+version: 0.2.0
```

Comment:
Newer TAC release available: 0.3.0

Reply:
It's at 0.4.0 since Sakari said the null plugin bumped it up to that, I'm not sure what it shows you 0.2.0?

Reply:
Ya, I realized that in the second commit. I was reviewing commit by commit, where it was 0.2.0 in the first commit.

Reply:
Oh ok, ill rebase once all the changes are approved.